docs/asciidoc/modules/ROOT/pages/overview/apoc.redis/apoc.redis.eval.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This file is generated by DocsTest, so don't change it!
+////
+
+= apoc.redis.eval
+:description: This section contains reference documentation for the apoc.redis.eval procedure.
+
```

Comment:
In the parameters provided to the procedure, `keys` are bound to the `KEYS[n]` like special array of the Lua script and `values` are bound to the `ARGV[n]` like special array of the Lua script.

Reply:
Might want to add this to the docs to try to make this procedure clearer.....at least to us non-Redis users, it feels a bit complicated. 😅

Reply:
You're right, it's not easily understandable. Now I've added this phrase into `@Description` and 2 usage examples in the documentation.